src/Menu/CachedBackendMenuBuilder.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Bolt\Menu;
+
+use Psr\SimpleCache\CacheInterface;
+use Symfony\Component\Stopwatch\Stopwatch;
+
+class CachedBackendMenuBuilder
```

Comment:
it's better to create common MenuInterface for autowiring (with getMenu())

Reply:
You'll have to elaborate on this, because I don't understand yet. :-)

Reply:
see https://symfony.com/doc/current/service_container/autowiring.html#working-with-interfaces